### PR TITLE
Remove `-b32` from bf rzil tests

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [X] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

This pr just removes `-b32` from the `bf` rzil tests. I don't think there's any good reason to have a 32-bit `bf` machine.

**Test plan**

All builds are green.

**Closing issues**

...

### CODE REVIEW

File: test/db/rzil/bf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 NAME=hello world
 FILE=bins/bf/hello-ok.bf
-ARGS=-b32 -eio.cache=1
+ARGS=-eio.cache=1
```
Comment (wargio) [inline test/db/rzil/bf:3]: is even io.cache needed? i think not.

Comment (kazarmy) [reply test/db/rzil/bf:3]: currently it's needed ... it shouldn't be needed though

Comment (thestr4ng3r) [reply test/db/rzil/bf:3]: It is needed for the emulation. The alternative is mapping a new malloc.

Comment (kazarmy) [reply test/db/rzil/bf:3]: Well if `oml` says the memory area is writable then i think it should be writable no matter the value of `io.cache`

**Review by wargio — APPROVED** [review]

**Pull request merged**